{"event": "verbose", "uuid": "5cf96c2d-902a-41cd-a444-ea886a2a2445", "counter": 7, "stdout": "\u001b[1;35mpackages/ansible/plugins/connection/vmware_tools.py) as it seems to be invalid:\u001b[0m", "start_line": 6, "end_line": 7, "runner_ident": "23ad5bfd-1e7a-420b-b245-f67a7f30f810", "created": "2023-05-16T06:00:04.516911"}